Argo/Extensions/RawRepresentable.swift

```diff
@@ -1,3 +1,7 @@
+/**
+  Default implementation of `Decodable` for `RawRepresentable` types using
+  `String` as the raw value.
```

Comment:
Since this is a very automatic function that results is less code, it might be good to write out an example:

If you have a type `State`:

```swift
enum State: String {
  case Unstarted = "unstarted"
  case Started = "started"
  case Completed = "completed"
}
```

which means the rawValue of each case is represented by a string (`"unstarted"`, `"started"`, `"completed"`), this allows Argo to automatically decode the data into types:

```swift
// code the developer has to write, if anything, that makes that work
```